### PR TITLE
Feature/pranit

### PR DESCRIPTION
**Pull request by pranitchandel**

### CODE REVIEW

File: src/pages/OverviewPage/InputFieldsSection/index.tsx [hunk 1]
```diff
@@ -36,8 +36,8 @@ const InputFiedsSection = () => {
             } = props;
             return (
               <>
-                <Row rowWidth={24}>
-                  <Col span={12}>
+                <Row spacing={2} spaceBetweenRow={5} justifyContent="flex-end">
```
Comment (AbhijeetKumarGupta) [inline src/pages/OverviewPage/InputFieldsSection/index.tsx:39]: please set it "flex-start" by default

**Review by AbhijeetKumarGupta — COMMENTED** [review]
please resolve comment

File: src/shared/atom/AlertSnackbar/index.tsx [hunk 2]
```diff
@@ -1,64 +1,76 @@
...
+      open={isOpen}
+      autoHideDuration={autoHideDuration}
+      onClose={handleClose}
+    >
+      <Alert onClose={handleClose} severity={alertType} sx={sx} {...alertProps}>
+        {message}
+      </Alert>
+    </Snackbar>
+  ) : (
+    <Alert style={sx} severity={alertType}>
```
Comment (AbhijeetKumarGupta) [inline src/shared/atom/AlertSnackbar/index.tsx:70]: please add alert props here also

Comment (pranitchandel) [reply src/shared/atom/AlertSnackbar/index.tsx]: done

File: src/pages/OverviewPage/InputFieldsSection/index.tsx (hunk 1 above)
Comment (pranitchandel) [reply src/pages/OverviewPage/InputFieldsSection/index.tsx]: done

**Review by lakDew — APPROVED** [review]
bhosadchod pehli bar me aad kiya kr

**Pull request merged**